signing_service/signing_service/tests/test_signing_service_methods.py

```diff
@@ -218,6 +219,25 @@ def test_that_argument_parser_should_fail_if_port_cannot_be_casted_to_int(self):
         with self.assertRaises(SystemExit):
             _parse_arguments()
 
+    def test_that_argument_parser_should_raise_exception_if_ethereum_key_cannot_be_decoded_to_string(self):
+        ethereum_private_key_raw = b'\x19\xf9\x16\xcc$F/\xf3\x1e\xdb\t8\xd6d\xb45\x14\x9c\x87e\xcc\xa6\x02\x17`\xb4Zj\xc9\x9c\xa3<'
+        try:
```

Comment:
I wanted to add proof that this particular key is causing the issue. Do you think it is not necessary?

Reply:
it's ok, just please add a comment explaining that in this block you are making sure that for the particular key, `UnicodeDecodeError` will be raised. BTW: **optionaly** you can use `with contextlib.suppress(UnicodeDecodeError)` instead of the `try ... except` - I find it more fancy and better intention revealing, but it is just personal preference  ;)

Reply:
:+1: OK comment added, I think `try/except` is more common so more readers will get it just like that.